core-web/libs/sdk/react/src/lib/models/index.ts

```diff
@@ -85,7 +87,8 @@ export interface DotCMSPageContext {
             forward: boolean;
         };
     };
-    isInsideEditor: boolean;
+    isInsideEditor?: boolean;
+    UVEState?: UVEState;
```

Comment:
Isn't this redundant?

Reply:
It is, but I dont want to mess up current implementations as we accorded. But maybe I can just remove it since we are going to publish it as `next`

Reply:
Duplicate the code, import from other place... this is the kind of things I want to avoid, I think it will be harder to follow up.

Reply:
No changes until we get there